### ISSUE TITLE
Docs: Add badges to docusauras Home.md

### ISSUE THREAD

**Issue by Amulya-coder** [anitab-org/bridge-in-tech-web#201]
### Is your feature request related to a problem? Please describe.

Add badges to the docusauras home.md

### Describe the solution you'd like

![image](https://user-images.githubusercontent.com/66437295/112351754-2b370d80-8cf0-11eb-8b21-ca3bf542673e.png)

### Additional context

None

**Comment by Rahulm2310** [comment]
@Amulya-coder Please remove the `fork` and `star` label from the screenshot as there is no need for them. Also, there is a `zulip` label already. Please remove that as well.

**Comment by mtreacy002** [comment]
I don't think we need "Maintained" and "PRs" tags either, @Amulya-coder. Plus, the License already shown on Github UI, so please remove that as it is redundant.

Saying that, I don't mind seeing all the GitHub UI tags + README tags being included in the docusaurus Home.md since we won't be able to see the GitHub UI tags there.
PS: @Rahulm2310 , while we're on the docusaurus topic, is it possible to use .md parser from the wiki .md files, that way we'll avoid the risk of having 2 different contents (wiki vs docusaurus). If we can use the .md parser in Home.md on docusaurus under the tags, that'll be best.

What do you all think?

**Comment by Rahulm2310** [comment]
> If we can use the .md parser in Home.md on docusaurus under the tags, that'll be best.
 
@mtreacy002 Unable to get your point. Can you be more clear

**Comment by mtreacy002** [comment]
@Rahulm2310, at the moment we have a copy of Home.md in the `docusaurus` docs folder (along with other .md files as well). 
<img width="467" alt="Screen Shot 2021-03-27 at 8 50 51 am" src="https://user-images.githubusercontent.com/29667122/112696240-8c6e0500-8ed9-11eb-8dd3-2fe7ccb9db20.png">

The issue with this, if the content in wiki page is modified, the changes will not be automatically detected in the docusaurus .md version which means we'll have conflicting information. If we can use the .md parser on our docusaurus, then the changes made to wiki will automatically applied to docusaurus. So, that's the first point I have in mind.

The second point is on the redundant tags that we don't want to have on README.md files since they are visible on Github UI. IMO, these tags are redundant on the README.md file for the repo, but they are important information that currently missing on the docusaurus Home.md. So, my suggestion is to add the tags in the docusaurus Home.md. 

I hope this helps explain my intention better 😉

**Comment by naveen8801** [comment]
Hi @mtreacy002 @Rahulm2310 @Amulya-coder  I'd like to work on it ,  can you assign it to me .?

**Comment by Amulya-coder** [comment]
@naveen8801 thanks for showing interest in this issue, firstly I give you the idea about this issue -> you have to add the badges on the docusauras Home.md so that it's visible on the docusauras documentation not on readme as explained by the <b>mtreachy</b> above ☝  would you know how to do that? if yes I will assign it to you.

**Comment by naveen8801** [comment]
@Amulya-coder  , i think you are talking about this https://github.com/anitab-org/bridge-in-tech-backend/tree/develop/docs/docs , and here Home.md should contain these badges or something else ?

**Comment by Amulya-coder** [comment]
Yes @naveen8801 would you like to work on this

**Comment by naveen8801** [comment]
Ya i'd like to work on it , but this means that i have to submit PR for ' bridge-in-tech-backend ' repo , right ?

**Comment by Amulya-coder** [comment]
Yes @naveen880, commented on that issue I will assign it to you.

**Comment by naveen8801** [comment]
Sure , i wanna work on this @Amulya-coder , 👍

**Comment by Rahulm2310** [comment]
@Amulya-coder I think this should be put on hold as we don't have the docusaurus website for frontend yet. Also, please update the issue description with what needs to be done.

**Comment by mtreacy002** [comment]
Making the issue available since now we have [BIT frontend docusaurus](https://bridge-in-tech-web.surge.sh/)

**Comment by IC1101Virgo** [comment]
Hello, can I work on this?

**Comment by Rahulm2310** [comment]
Sure @IC1101Virgo. Assigning you ☺️

**Comment by IC1101Virgo** [comment]
I have a doubt regarding the task what needs to be done? 
I can see the badges here https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/docs/docs/Home.md 
Do I need to add any more? Or need to add them somewhere else?

Thank you in advance

**Comment by Rahulm2310** [comment]
@IC1101Virgo you need to add the badges here on this repo i.e. `bridge-in-tech-web`. The link you are using is for the backend repo.

**Comment by harsh7in** [comment]
@Rahulm2310, I want to contribute to the project, can i work on this issue?

**Comment by isabelcosta** [comment]
@harsh-9in will assign to you since there are no updates for a while from @IC1101Virgo 

@IC1101Virgo if I got this wrong and you are still interested in contributing, let us know :)

**Comment by Esther-Wavinya** [comment]
Hello. Is this issue still open? @isabelcosta @Amulya-coder . If yes. Can I work on it?
